xcompile-all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+source $GOPATH/src/github.com/davecheney/golang-crosscompile/crosscompile.bash
```

Comment:
Yea I've found myself including this in all my repos, which probably isn't the best way to manage it. It's super nice to be able to cross-compile to whatever platform I want.